core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -1040,17 +1044,23 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
   test("reserve/release unroll memory") {
     store = makeBlockManager(12000)
     val memoryStore = store.memoryStore
+    val dummyBlock = TestBlockId("")
```

Comment:
ok, I wanted to avoid making an array buffer every time we call that, but I guess it's not that expensive.